add(epp): mobotix snmp - procedure v2 (#791)
centreon/centreon-documentation: en/integrations/plugin-packs/procedures/hardware-devices-camera-mobotix-snmp.md

| *interface\_name*\#interface.traffic.out.bitspersecond | Outgoing traffic going through the interface        | b/s  |
| *interface\_name*\#interface.traffic.out.percentage    | Percentage of the interface's *out* bandwidth usage | %    |

A regexp filter is available to target a specific interface identifier - ifName [```--interface='^eth0$' --name```]

<!--System-->

| Metric name                         | Description                      | Unit |
| :---------------------------------- | :------------------------------- | :--- |
| system.sdcard.usage.percent         | SD card usage                    | %    |
| system.temperature.internal.celsius | Internal temperature             | C    |
| system.temperature.external.celsius | External temperature             | C    |
| system.temperature.gps.celsius      | GPS temperature                  | C    |
| system.illumination.right.lux       | Illumination of the right sensor | lx   |
| system.illumination.left.lux        | Illumination of the left sensor  | lx   |
| system.video.framerate.persecond    | Current video framerate          |      |

<!--END_DOCUSAURUS_CODE_TABS-->

## Prerequisites

> To control your Mobotix, the SNMP must be configured.

Reviewer: ```suggestion
To monitor your Mobotix, the SNMP must be configured.

The Poller should be able to perform requests against the Mobotix device over SNMP UDP/161 port. 
```

## Setup

<!--DOCUSAURUS_CODE_TABS-->

<!--Online IMP Licence & IT-100 Editions-->

1. Install the Centreon Plugin on every Poller:

```bash
yum install centreon-plugin-Hardware-Devices-Camera-Mobotix-Snmp
```

> 2. On the Centreon Web interface in "Configuration > Plugin packs > Manager", install the *Mobotix Camera* Plugin Pack

Reviewer: ```suggestion
2. On the Centreon Web interface in "Configuration > Plugin Packs > Manager", install the *Mobotix Camera* Plugin Pack
```

<!--Offline IMP License-->

1. Install the Centreon Plugin on every Poller:

```bash
yum install centreon-plugin-Hardware-Devices-Camera-Mobotix-Snmp
```

2. On the Centreon Central server, install the Centreon Plugin Pack from the RPM:

```bash
yum install centreon-pack-hardware-devices-camera-mobotix-snmp
```

> 3. On the Centreon Web interface in "Configuration > Plugin packs > Manager", install the *Mobotix Camera* Plugin Pack

Reviewer: ```suggestion
3. On the Centreon Web interface in "Configuration > Plugin Packs > Manager", install the *Mobotix Camera* Plugin Pack
```

<!--END_DOCUSAURUS_CODE_TABS-->

## Host configuration

* Add a new Host and fill the *IP Address/FQDN*, *SNMP Version* and *SNMP Community* fields according to the device's configuration
* Apply the *HW-Device-Camera-Mobotix-SNMP-custom* Host Template

> When using SNMP v3, use the SNMPEXTRAOPTIONS Macro to add specific authentication parameters

| Mandatory | Name             | Description                                    |
| :-------- | :--------------- | :--------------------------------------------- |
|           | SNMPEXTRAOPTIONS | Configure your own SNMPv3 credentials combo    |

## How to test the Plugin and what are the main options for?

> Once the plugin installed, log into your Centreon Poller CLI using the *centreon-engine* user account

Reviewer: ```suggestion
Once the Plugin installed, log into your Centreon Poller CLI using the *centreon-engine* user account
```
